scripts/make-rules/golang.mk

```diff
@@ -18,4 +18,8 @@ go.cover:
 .PHONY: go.test
 go.test:
 	@echo "===========> Run go test"
-	@go test -coverprofile=knetty_coverage.out ./...
\ No newline at end of file
+	@go test -coverprofile=knetty_coverage.out ./...
+
+go.changelog:
```

Comment:
Please add `.PHONY: go.changelog`.

U can refer to this [link](https://www.google.com.hk/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwj52OuU2MH8AhU5-HMBHfDhBZQQFnoECA0QAw&url=https%3A%2F%2Fftp.gnu.org%2Fold-gnu%2FManuals%2Fmake-3.79.1%2Fhtml_node%2Fmake_34.html%23%3A~%3Atext%3DA%2520phony%2520target%2520is%2520one%2Cname%252C%2520and%2520to%2520improve%2520performance.&usg=AOvVaw2DuJ6A18S4Yizi3mC_EWAf) for more info.

Reply:
fixed